client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -489,6 +491,7 @@ function StatsCommercialFlowOptOutForm( {
 					</Button>
 				) }
 				{ ( ! supportsOnDemandCommercialClassification ||
+					! isCommercial ||
```

Comment:
The button is also the Continue button, which we don't want to hide for non-commercial users.

Reply:
Line 315 prevents this from showing up for all new sites.

Reply:
Sorry, I mixed the testing instructions.